.github/workflows/ci.yaml

```diff
@@ -305,14 +305,18 @@ jobs:
         if: needs.changes.outputs.test_full_suite == 'true'
         run: |
           . venv/bin/activate
-          pre-commit run --hook-stage manual flake8 --all-files
+          pre-commit run --verbose --hook-stage manual flake8 --all-files
       - name: Run flake8 (partially)
         if: needs.changes.outputs.test_full_suite == 'false'
         shell: bash
         run: |
           . venv/bin/activate
           shopt -s globstar
-          pre-commit run --hook-stage manual flake8 --files {homeassistant,tests}/components/${{ needs.changes.outputs.integrations_glob }}/**/*
+          pre-commit run --verbose --hook-stage manual flake8 --files {homeassistant,tests}/components/${{ needs.changes.outputs.integrations_glob }}/**/*
+      - name: Unregister flake8 problem matcher
+        run: |
+          echo "::remove-matcher owner=flake8-error::"
+          echo "::remove-matcher owner=flake8-warning::"
```

Comment:
Are we having issues with this that is needs to be changed?
There is nothing running in this job after this statement either?

The same applies to all other occurances.

Reply:
I ran into a thing where the one overly broad pattern matcher matched stuff later.

It's polite to drop marchers that are no longer relevant especially if there's a risk of them being silly.

That said, with the fix to the overly broad matcher, I'm happy to drop this.

Reply:
> I ran into a thing where the one overly broad pattern matcher matched stuff later.

Could you please provide the case in the changes added that are fixed? I don't see the issue, as the changes added have no follow-up steps after the part where they are removed.

Reply:
unresolved this review comment, as the question hasn't been answered. Could you please clarify the issue you ran into?